packages/snaps-execution-environments/src/common/endowments/endowments.test.browser.ts

```diff
@@ -25,6 +25,7 @@ lockdown({
   domainTaming: 'unsafe',
   errorTaming: 'unsafe',
   stackFiltering: 'verbose',
+  overrideTaming: 'severe',
```

Comment:
what's this change for?

Reply:
To mirror the LM/SES config we run the execution environments under: https://github.com/MetaMask/snaps/blob/main/.yarn/patches/%40lavamoat-lavapack-npm-5.4.1-2548b3e225.patch

I believe Maarten was getting an error in tests but not when actually using the execution environments because of this mismatch

Reply:
I think we could revert this now, since we removed the `SnapError` endowment later on, but regardless it's good for the config we use here to match LavaMoat.